src/index.js

```diff
@@ -0,0 +1,64 @@
+import './style.css';
+import { saveLocalStorage, updateCompleted } from './active.js';
+import {
+  editTodo, removeTodos, addTodo, clearCompletedTodos,
+} from './add.js';
+
+let itemsArray = [];
+
+const storageData = JSON.parse(localStorage.getItem('toDoStorage'));
+if (storageData) {
+  itemsArray = storageData;
+}
```

Comment:
- [ ] Kindly It could be better if you make use of the short circuit evaluation 

![999](https://user-images.githubusercontent.com/79058364/135081169-8d850f53-f466-4b15-a01f-abcac68c7746.png)

This is to ensure your code follows the Javascript best practices. instead of trying to use this approach:
> `let storageData = JSON.parse(localStorage.getItem('toDoStorage')) || [ ];`

Reply:
Hello @BenMukebo,
Thank you for your feedback.